pkg/generators/forwarding/fluentd/output_conf_forward_test.go

```diff
@@ -43,7 +44,7 @@ var _ = Describe("Generating fluentd secure forward output store config blocks",
 	   @type forward
 	   <security>
 	     self_hostname "#{ENV['NODE_NAME']}" 
-	     shared_key secureforward-receiver
+	     shared_key "#{File.open('/var/run/ocp-collector/secrets/my-infra-secret/shared_key') do |f| f.readline end.rstrip}"
```

Comment:
is it going to be a problem if you want to run the test and you have to use `sudo` to `mkdir -p /var/run/ocp-collector/secrets/my-infra-secret/`?

Reply:
@richm Who has to `sudo mkdir`? The secret is mounted into the pod via the deployment so the container process should already have permissions to read the content of the file

Reply:
If this test will always be run inside the container, then correct.

I know we've had issues in the past with tests that required you to do things like `sudo mkdir /usr/share/logging` when running the test on your local machine outside of a container.  Maybe that doesn't apply to this test.

Reply:
@richm to follow up on our slack convo here this code is only evaluated when the fluent pod starts.  There are no tests outside the container that would eval this.

Reply:
The answer is - this test assumes it will always be running in a fluentd deployed by cluster logging